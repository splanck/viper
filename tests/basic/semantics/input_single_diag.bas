10 INPUT A
20 PRINT A
30 END
