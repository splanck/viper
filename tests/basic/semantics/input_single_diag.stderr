Trap @main#1 line 1: DomainError (code=0): INPUT: expected numeric value
